api/extension-guides/chat.md

```diff
@@ -269,6 +269,77 @@ cat.followupProvider = {
 
 > **Tip:** Follow-ups should be written as questions or directions, not just concise commands.
 
+### Implement participant detection
+
+To make it easier to use chat participants with natural language, you can implement participant detection. Participant detection is a way to automatically route the user's question to a suitable participant, without having to explicitly mention the participant in the prompt. For example, if the user asks "How do I optimizer this query?", the question would be automatically routed to the `@database` participant.
```

Comment:
```suggestion
To make it easier to use chat participants with natural language, you can implement participant detection. Participant detection is a way to automatically route the user's question to a suitable participant, without having to explicitly mention the participant in the prompt. For example, if the user asks "How do I optimize this query?", the question would be automatically routed to the `@database` participant.
```

There is no builtin `@database` participant, so this paragraph is a bit misleading. Can we pick one of our builtin participants as the poster child instead?